__tests__/config/helpers/accountResolver.ts

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable no-console */
+import { GS_DEFAULT_TEST_PROVIDER_URL } from '../constants';
+
+class AccountResolver {
+  get providedUrl() {
+    return process.env.TEST_PROVIDER_BASE_URL || process.env.TEST_RPC_URL;
+  }
+
+  get hasAllAccountEnvs() {
+    return process.env.TEST_ACCOUNT_ADDRESS && process.env.TEST_ACCOUNT_PRIVATE_KEY;
+  }
+
+  get hasPartialAccountEnvs() {
+    return process.env.TEST_ACCOUNT_ADDRESS || process.env.TEST_ACCOUNT_PRIVATE_KEY;
+  }
+
+  private async fetchAccount(url: string) {
+    const response = await fetch(`${url}predeployed_accounts`);
+    const [account] = await response.json();
+    const { address, private_key, initial_balance } = account;
+    process.env.TEST_ACCOUNT_ADDRESS = address;
+    process.env.TEST_ACCOUNT_PRIVATE_KEY = private_key;
+    process.env.INITIAL_BALANCE = initial_balance;
+  }
+
+  private async isAccountSet(isRsDevnet: boolean): Promise<boolean> {
```

Comment:
```suggestion
  private async isAccountSet(isDevnet: boolean): Promise<boolean> {
```

...I think just devnet without the rs is ok!